mbofui/app/app.js

```diff
@@ -0,0 +1,224 @@
+var bofDataRef = new Firebase( 'https://flickering-fire-3313.firebaseio.com/bofs' );
+app.controller( 'mapController', [ '$scope', '$filter', '$timeout', '$log', 'leafletData', function ( $scope, $filter, $timeout, $log, leafletData ) {
+  angular.extend( $scope, {
+    center: {
+      zoom: 16,
+      autoDiscover: true
+    },
+    events: {},
+    controls: {
+      draw: {}
+    },
+    layers: {
+      baselayers: {
+        mapbox_light: {
+          name: 'Mapbox Light',
+          url: 'https://api.tiles.mapbox.com/v4/{id}/{z}/{x}/{y}.png?access_token={accessToken}',
+          type: 'xyz',
+          layerOptions: {
+            id: 'gsilver.pk8alhme',
```

Comment:
Rather than using your name, make an MCommunity group for the project and use its email address or name when you register a token.  If you use your name, what happens when you get hit by a beer truck and you're not able to come support the app, get a new token, etc.?